lib/smart_answer_flows/marriage-abroad/outcomes/os_affirmation/_fees.govspeak.erb

```diff
@@ -11,16 +11,9 @@
                locals: { calculator: calculator } %>
   <% end %>
 
-  <% if calculator.ceremony_country == 'finland' %>
-    <%= render partial: 'pay_in_euros_or_visa_electron.govspeak.erb' %>
-  <% elsif calculator.ceremony_country == 'philippines' %>
-    <%= render partial: 'pay_by_cash_or_bank_cheque' %>
-  <% elsif calculator.ceremony_country == 'norway' %>
-    <%= render partial: 'pay_by_visas_or_mastercard.govspeak.erb' %>
-  <% elsif calculator.ceremony_country == 'south-korea' %>
-    <%= render partial: 'pay_by_cash_in_local_currency_or_credit_card_not_cheque' %>
-
-  <% else %>
-    <%= render partial: 'pay_by_cash_or_credit_card_no_cheque.govspeak.erb' %>
+  <% if calculator.services_payment_partial_name.present? %>
+    <% partial = calculator.services_payment_partial_name + '.govspeak.erb' %>
+    <%= render partial: partial,
+               locals: { calculator: calculator } %>
```

Comment:
Very minor: I realise you're probably trying to use the same style as other code in this template, but I think they are the relics of lines that were previously longer. I'd prefer that this call to `render` was on a single line, but I understand if you'd prefer to address it separately.